itez/beneficiary/models.py

```diff
@@ -1,42 +1,193 @@
-from django.db import models
-from django.db.models.deletion import PROTECT 
+from django.contrib.gis.db import models
+from django.db.models.fields.related import create_many_to_many_intermediary_model
 from django.utils.translation import gettext_lazy as _
 
+from .utils import generate_uuid_and_agent_code
+
+
+class Beneficiary(models.Model):
+    """
+    Implements the Beneficiary properties and required methods.
+    """
+
+    GENDER = (
+        ("male", _("Male")),
+        ("female", _("Female")),
+        ("other", _("Other"))
+    )
+
+    MARITAL_STATUS = (
+        ("single", _("Single")),
+        ("married", _("Married")),
+        ("seperated", _("Seperated")),
+        ("divorced", _("Divorced")),
+        ("widowed", _("Widowed")),
+    )
+
+    TERTIARY_EDUCATION_QUALIFICATION = (
```

Comment:
we also want to cover basic primary and secondary O'Leve education optoins and NONE for no formal school